Tests/LoggerMessageTests.swift

```diff
@@ -174,27 +174,27 @@ class SynchronousLoggerMessageLogLevelTestCase: SynchronousLoggerTestCase {
         let message = TestMessage()
 
         // When
-        log.debug { () -> LogMessage in
+        log.debug {
             log.debug { message }
             return message
         }
```

Comment:
This is really the core change in this PR. If anyone can find a workaround for this that doesn't require declaring the return type, then beers are on me. Best idea so far is @AtomicCat's idea to make `String` conform to `LogMessage`. Problem with that is that it would expose the `name` and `attributes` properties publicly on `String` which is just something we shouldn't do. We both agreed that the approach used in this PR is better than that approach.